index.bs

```diff
@@ -1419,18 +1423,13 @@ interface XRRigidTransform {
 
 The <dfn constructor for="XRRigidTransform">XRRigidTransform(|position|, |orientation|)</dfn> constructor MUST perform the following steps when invoked:
 
-  1. Let |transform| be a new {{XRRigidTransform}}.
-  1. If |position| is not a {{DOMPointInit}} initialize |transform|'s {{XRRigidTransform/position}} to <code>{ x: 0.0, y: 0.0, z: 0.0, w: 1.0 }</code>.
+  1. Let |transform| be a [=new=] {{XRRigidTransform}} in the [=current realm=].
+  1. Let |transform|'s {{XRRigidTransform/position}} be a [=new=] {{DOMPointReadOnly}} in the [=current realm=].
```

Comment:
Was worried that we lost something in this section, but since the constructor defaults line up with what we had originally specified this is a nice cleanup. Thanks!